release_notes.md

```diff
@@ -1,55 +1,84 @@
-    - Subnet - Text Field
-    - Firmware - Text Field
-    - Vendor - Text Field
-    - Product - Text Field
-    - Asset - ManyToMany
-    - Protocol - Picklist
-    - ESPZone - Text Fields
-    - Facility - Text Fields
-    - Level - Picklist
-    - Zone - Picklist
-    - Asset Type - Picklist
-    - Vulnerability Risk Status - Picklist
-
-- Moved following picklist from Asset from this Solution Pack to **Soar Framework Solution Pack v2.2.0**:
-    - AssetLevel 
-    - AssetProtocol
-    - AssetType
-    - AssetZone
-    - VulnerabilityRiskStatus
-
-- Removed playbook `MITRE ATT&CK > Fetch Latest Data` from collection `02 - Use Case - IT OT Asset Management` as MITRE data can be ingested using data ingestion from MITRE connector.
-- Removed playbook `Create Alerts Record`
-- Removed playbook `Alert - Escalate To Incident`, `Alert - Escalate to Incident (Link Relations)`, and `Alert - Escalate To Incident (No Trigger)`, as implemented it in **Soar Framework Solution Pack v2.1.1** 
-- Moved `MITRE Widget` from dashboard **Asset Risk Overview** to **MITRE ATT&CK Enrichment Framework v2.2.0**
\ No newline at end of file
+- Dashboards **Asset Change Activity Tracking**, **Asset Overview**, and **Asset Risk Overview** can now be assigned to various roles; the role assignment was not possible earlier.
+
+- Following playbook collection names were changed to be inline with other playbook collections in FortiSOAR&trade;:
+    - **00 - Use Case - Asset Change Activity** renamed to **02 - Use Case - Asset Change Activity**.
+    - **00 - Use Case - Asset Management** renamed to **02 - Use Case - IT OT Asset Management**.
+
+## Deprecations
+
+- Following fields from the **Asset** module in this solution pack are now a part of **SOAR Framework** v2.2.0 and later:
+
+    | Field Name                | Input Type  |
+    |:--------------------------|:------------|
+    | Network                   | Text Field  |
+    | Subnet                    | Text Field  |
+    | Firmware                  | Text Field  |
+    | Vendor                    | Text Field  |
+    | Product                   | Text Field  |
+    | Asset                     | ManyToMany  |
+    | Protocol                  | Picklist    |
+    | ESPZone                   | Text Fields |
+    | Facility                  | Text Fields |
+    | Level                     | Picklist    |
+    | Zone                      | Picklist    |
+    | Asset Type                | Picklist    |
+    | Vulnerability Risk Status | Picklist    |
+
+- Following picklists from the **Asset** module in this solution pack are now a part of **SOAR Framework** v2.2.0 and later:
+    - `AssetLevel`
+    - `AssetProtocol`
+    - `AssetType`
+    - `AssetZone`
+    - `VulnerabilityRiskStatus`
+
+- The playbook *MITRE ATT&CK > Fetch Latest Data* from the collection **02 - Use Case - IT OT Asset Management** is now deprecated as the MITRE data can be ingested using data ingestion from MITRE connector.
+
```

Comment:
Connector is to be renamed from "MITRE" to "MITRE ATT&CK"

Reply:
Fixed